apps/course-authoring/src/renderer/pages/editor/structure/index.tsx

```diff
@@ -1,14 +1,92 @@
-import React from 'react';
+import React, { useState } from 'react';
 import * as style from './styles.module.scss';
 import { Default as Nav } from '@owlui/navigationdrawer';
+import { Default as Btn } from '@owlui/button';
+import { CourseData } from '../course.types';
+import { CourseTemplate } from '../templates/course';
 
 export const PageRoute = '/structure';
 export const PageName = 'Structure';
 
 export const PageElement = () => {
+  const [courseData, setCourseData] =
+    React.useState<CourseData>(CourseTemplate);
+
+  const handleChange = (
+    e: React.ChangeEvent<
+      HTMLInputElement | HTMLTextAreaElement | HTMLSelectElement
+    >
+  ) => {
+    const newData = { ...courseData };
+
+    newData[e.target.name as keyof typeof courseData] = e.target.value;
+
+    setCourseData(newData);
+  };
+
+  const SidebarContent = (
+    <>
+      <div>
+        <h3>Project Settings</h3>
+        <div className="field">
```

Comment:
Each field will be replaced by its proper owlUI component once we get them created.